frontend/src/components/Table.tsx

```diff
@@ -113,10 +114,19 @@ function Table(props: Props) {
                 scope="col"
                 {...column.getHeaderProps()}
                 style={
-                  props.selection !== "none" ? { padding: "0.5rem 1rem" } : {}
+                  props.selection !== "none"
+                    ? { padding: "0.5rem 1rem" }
+                    : { minWidth: column.minWidth }
                 }
               >
-                {column.render("Header")}
+                <span>
+                  {typeof column.render("Header") === "string"
+                    ? (column.render("Header") as string)
+                        .split('"')
+                        .filter(Boolean)
+                        .join()
+                    : column.render("Header")}
```

Comment:
Can you add some comments to explain why the `split`, `filter` and `join` are needed. I'm worried that we may forget in the future why are we doing this.